server/server.go

```diff
@@ -101,3 +101,22 @@ func (s *Server) GetEntries(id string) (*[]Entry, error) {
 	}
 	return &l, nil
 }
+
+// SumCalories : return sum of calories from entries for specific user
+func (s *Server) SumCalories(id string) (int, error) {
+	var sum int
+	sqlStatement := `
+	SELECT SUM(calories)
+	FROM entries
+	WHERE fuser_id = $1`
+	rows, err := s.db.Query(sqlStatement, id)
+	if err != nil {
+		log.Print("No user to count calories for" + err.Error())
+		return 0, nil
+	}
+	defer rows.Close()
+	for rows.Next() {
+		rows.Scan(&sum)
```

Comment:
variable declarations should be close to where they are used, would probably be a good idea to change this to:
```go
// ... other stuff
var sum int
defer rows.Close()
for rows.Next() {
    rows.Scan(&sum)
}
// ... more stuff
```